### PR TITLE
[PORT-134] Feature/map ts a

### PR DESCRIPTION
**Pull request by adamdoe**
## Checklist before requesting a review
- [ ] My pull request was branched from and targets the test branch
- [ ] I have performed a self-review of my code
- [ ] I have manually tested all packages affected (bonus points for automations)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have checked to ensure there aren't other open pull requests for the same change

## Did you test your feature in the following environments?
- [ ] Standalone Component
- [ ] Standalone Full Editor
- [ ] CDC Internal Checks

### CODE REVIEW

File: eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (joshlacey) [inline eslintrc.json:1]: @adamdoe why did you add this file?

Comment (adamdoe) [reply eslintrc.json:1]: I had branched from test before the typescript changes made it in, and wanted to copy the same format.

**Comment by adamdoe** [comment]
@joshlacey - nice ts, helpers, and general improvements here. I don't see anything that concerns me. Are you done with your work on this PR?
